HPXMLtoOpenStudio/resources/misc_loads.rb

```diff
@@ -279,25 +279,63 @@ def self.apply_pool_or_permanent_spa_pump(runner, model, pool_or_spa, conditione
     mel.setSchedule(pump_sch)
   end
 
-  # TODO
+  # Returns the default residual miscellaneous electric (plug) load energy use
+  # and sensible/latent fractions.
   #
   # @param cfa [Double] Conditioned floor area in the dwelling unit (ft2)
-  # @return [TODO] TODO
-  def self.get_residual_mels_default_values(cfa)
-    annual_kwh = 0.91 * cfa
+  # @param num_occ [Double] Number of occupants in the dwelling unit
+  # @param unit_type [String] HPXML::ResidentialTypeXXX type of dwelling unit
+  # @return [Array<Double, Double, Double>] Plug loads annual use (kWh), sensible fraction, and latent fraction
+  def self.get_residual_mels_default_values(cfa, num_occ, unit_type)
+    if num_occ.nil? # Asset calculation
+      # ANSI/RESNET/ICC 301
+      annual_kwh = 0.91 * cfa
+    else # Operational calculation
+      # RECS 2020
+      if unit_type == HPXML::ResidentialTypeSFD
+        annual_kwh = 786.9 + 241.8 * num_occ + 0.33 * cfa
+      elsif unit_type == HPXML::ResidentialTypeSFA
+        annual_kwh = 654.9 + 206.5 * num_occ + 0.21 * cfa
+      elsif unit_type == HPXML::ResidentialTypeApartment
+        annual_kwh = 706.6 + 149.3 * num_occ + 0.10 * cfa
+      elsif unit_type == HPXML::HPXML::ResidentialTypeManufactured
+        annual_kwh = 1795.1 # No good relationship found in RECS, so just using a constant value
```

Comment:
Bringing attention to this. If anyone has a better idea, I'm all ears.

Reply:
I guess I am not surprised by the lack of relationship for `num_occ` and `cfa`. What about the number of bedrooms? I know it is a little strange to introduce a separate variable just for Manufactured homes. I do like the idea of the value changing based on some variable and not constant, but if we don't have a clear trend then maybe we should keep it constant.

Reply:
@joseph-robertson's analysis shows that all the different combinations of independent variables (CFA, # bedrooms, # occupants) have very low R2 values for MH. Worse, all the coefficients are negative. So, for example, the equation based on # bedrooms would be:
`annual_kwh = 2299.72 - 191.7 * num_br`
and would have lower electricity use when there are more bedrooms. Which I don't think makes sense.